### PR TITLE
Added a noise parameter to GenerateDatasets

### PR DESCRIPTION
**Pull request by Shivesh777**
The methods now allow a noise value as an integer to add random points into the dataset.

### CODE REVIEW

**Review by Ford-Robert — DISMISSED** [review]
Excellent implementation for adding the noise to our datasets at low cost

File: src/main/java/com/playground/playground/GenerateDatasets.java [hunk 1]
```diff
@@ -34,13 +36,25 @@ public static ArrayList<ArrayList<ArrayList<Double>>> generateCircular() {
             x1.add(xp);
             y1.add(yp);
         }
+        // Add extra random points (noise)
+        for (int i = 0; i < noise * 10; i++) {
```
Comment (Rishit-dagli) [inline src/main/java/com/playground/playground/GenerateDatasets.java:40]: WHat si the usual range of x,y if we have no noise?

**Review by Rishit-dagli — COMMENTED** [review]
Just a minor would be seeing if we do not deviate current range by a lot with any added noise, but other than that this looks good

File: src/main/java/com/playground/playground/GenerateDatasets.java (hunk 1 above)
Comment (Shivesh777) [reply src/main/java/com/playground/playground/GenerateDatasets.java:40]: It is -10 to +10

Comment (Shivesh777) [reply src/main/java/com/playground/playground/GenerateDatasets.java:40]: Do you want to reduce the noise range from 50 to 30? That should be done in the UI.

**Review by snkrs23 — DISMISSED** [review]

**Review by Ford-Robert — APPROVED** [review]
We should merge these changes asap!

**Review by Rishit-dagli — APPROVED** [review]
Ready to merge now /approve

**Pull request merged**